enemies/Enemy.gd

```diff
@@ -0,0 +1,14 @@
+extends CharacterBody2D
+
+const SPEED = 100
+var player
```

Comment:
Add the type here:

```suggestion
var player: Player
```

And you need to add: `class_name Player` in the `Player` Script

Reply:
You wanted types, so add types every here! 😈

Reply:
In the `const SPEED = 100`, I think it's not needed but, well, I leave you the choice

Reply:
it has to be `var player: CharacterBody2D` since in the enemy script it doesnt have access to the player directly